parlai/core/torch_ranker_agent.py

```diff
@@ -173,6 +181,9 @@ def __init__(self, opt, shared=None):
                 self.model, device_ids=[self.opt['gpu']], broadcast_buffers=False
             )
 
+    def build_loss(self, opt):
+        return nn.CrossEntropyLoss(reduce=True, size_average=False)
```

Comment:
it would be vaguely nice if this were made more similar to `compute_loss` in TGA, which is nice and easy to override with custom losses. If not that, can we make this `_build_loss` for the time being so it's less likely someone overrides it? I'm hoping we can unify those two interfaces.

Reply:
reverting this change and moving to new PR